library/core/src/intrinsics.rs

```diff
@@ -2277,7 +2277,8 @@ extern "rust-intrinsic" {
     ///
     /// # Safety
     ///
-    /// It's UB to call this if any of the *bytes* in `*a` or `*b` are uninitialized.
+    /// It's UB to call this if any of the *bytes* in `*a` or `*b` are uninitialized or carries a
```

Comment:
Minor English grammar nit, since `are` is plural, too, for consistency, it shouldn’t be `carries`, I think. (I’m not a native speaker though.)

```suggestion
    /// It's UB to call this if any of the *bytes* in `*a` or `*b` are uninitialized or carry a
```